Lib/fontTools/feaLib/lexer.py

```diff
@@ -231,7 +231,11 @@ def make_lexer_(file_or_path, location=None):
             try:
                 fileobj = open(filename, "r", encoding="utf-8")
             except IOError as err:
-                raise FeatureLibError(str(err), location)
+                # FileNotFoundError does not exist on Python < 3.3
+                import errno
+                if err.errno == errno.ENOENT:
```

Comment:
Does this work on Windows? (Might be good to add a test case that triggers this error, then we’ll know for sure).

Reply:
yes, it works on Windows too.

we do have a test that asserts that a FeatureLibError (superclass of IncludedFeaNotFound) is raised if we `include blah` (non existent file), and the test passed on Windows, and I can see the line covered in the coverage.

I can make the test more specific if you like.

Reply:
it works on windows too, see
https://ci.appveyor.com/project/fonttools/fonttools/build/1.0.1535